scss/components/_actor-sheet.scss

```diff
@@ -159,14 +186,16 @@
 .actor-sheet table tr.sticky {
     position: sticky;
     top: 0;
-    z-index: 1; /* pesky checkboxes */
-    background-color: #cfcec3;
+    z-index: 1;
+    /* pesky checkboxes */
```

Comment:
I think that this comment is still intended to be with the `z-index`? Looks like this comment might apply in at least 1 other place.

Reply:
Editor somehow autoformatted these and I didn't catch that :)